integration/tests/cook/util.py

```diff
@@ -232,11 +232,14 @@ def wait_until_inner():
         raise
 
 
-def all_instances_done(response, accepted_states=['success', 'failed']):
+def all_instances_done(response, accepted_states=None):
```

Comment:
Why did this need to change? I don't like the double defaulting.

Reply:
If this is an issue with the default list being mutable, I'd prefer changing the list to a tuple rather than switching to None. I like having the default value in the signature.

Reply:
Yes, the change was because the default shouldn't be mutable. Happy to change it to a tuple instead.

Reply:
I'm good with using tuples.

Reply:
This was taken care of in a separate PR